dsl/src/commonMain/kotlin/it/unibo/collektive/aggregate/api/operators/AggregateOperators.kt

```diff
@@ -24,7 +23,9 @@ import it.unibo.collektive.field.Field
  * In this case, the field returned has the result of the computation as local value.
  */
 fun <ID : Any, Scalar> Aggregate<ID>.neighboringViaExchange(local: Scalar): Field<ID, Scalar> =
-    exchange(local, ::identity)
+    exchanging(local) { toYield ->
+        toYield.map { local }.yielding { toYield }
```

Comment:
```suggestion
        toYield.mapToConstantField { local }.yielding { toYield }
```

This change must me made after #225 is merged and this branch is rebased with it.